File: c-cs50-harvard-ifood/exercicio2/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    
    // Solicitar altura da pirâmide ao usuário
    do
    {
        height = get_int("Altura: ");
    }
    while (height < 1 || height > 8);
    
    // Gerar as duas pirâmides
    for (int row = 1; row <= height; row++)
    {
        // Espaços antes da pirâmide esquerda
        for (int space = 0; space < height - row; space++)
        {
            printf(" ");
        }
        
        // Hashes da pirâmide esquerda
        for (int hash = 0; hash < row; hash++)
        {
            printf("#");
        }
        
        // Espaço entre as pirâmides
        printf("  ");
        
        // Hashes da pirâmide direita
        for (int hash = 0; hash < row; hash++)
        {
            printf("#");
        }
        
        // Nova linha
        printf("\n");
    }
}

File: python-cc50-harvard-ifood/exercicio1/mario.py
def main():
    while True:
        try:
            height = int(input("Height: "))
            if 1 <= height <= 8:
                break
        except ValueError:
            pass

    for i in range(1, height + 1):
        # Espaços à esquerda
        print(" " * (height - i) + "#" * i)

if __name__ == "__main__":
    main()
